tests/tdastro/sources/test_sncosmo_models.py

```diff
@@ -18,6 +19,18 @@ def test_sncomso_models_hsiao() -> None:
     assert np.allclose(fluxes, [4.31210900e-20, 7.46619962e-20, 1.42182787e-19])
```

Comment:
I think I missed this one last time I reviewed it. The example on the sncosmo site has set the model t0 = 55000 but we have t0=0 (so we will get zero fluxes) and they shouldn't match. The reason the test passed is because the flux values are very small. The default threshold for np.allclose is 1e-8

Reply:
Also the example is setting z=0.5, so we would not get a matched flux without the redshift effect.